R/utils-fetch.R

```diff
@@ -305,15 +305,6 @@ fetchQualifiers <- function(reportObject){
   return(val)
 }
 
-#'Fetch Qualifier Metadata (TSS)
```

Comment:
This should have been removed when we merged the TSS work in. It became redundant. It's removed now :D